Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -1296,6 +1445,68 @@ script:
       description: 'A CSV list of DB cluster snapshot attributes to add to the attribute specified by AttributeName.'
       required: false
 
+  - name: aws-rds-db-cluster-snapshot-set-to-private-quick-action
+    description: Revokes public access to the RDS DB cluster snapshot, making it private by running the aws rds modify-db-cluster-snapshot-attribute command. Successful execution requires the rds:ModifyDBClusterSnapshotAttribute permission.
```

```suggestion
    description: Revokes public access to the RDS DB cluster snapshot, making it private by running the aws rds modify-db-cluster-snapshot-attribute command.
```
Replace aws rds modify-db-cluster-snapshot-attribute.